src/dswx_sar/mosaic_rtc_burst.py

```diff
@@ -763,25 +764,45 @@ def run(cfg):
         epsg_list = []
 
         for ind, input_dir in enumerate(input_list):
-            # find HDF5 metadata
-            metadata_path = glob.glob(f'{input_dir}/*h5')[0]
-            epsg_list.append(read_metadata_epsg(metadata_path)['epsg'])
 
-        epsg_output = majority_element(epsg_list)
+            first_rtc_path_iter = glob.iglob(f'{input_dir}/*_{first_pol}*.tif')
```

Comment:
```suggestion
            first_rtc_path_iter = glob.iglob(f'{input_dir}/*_{first_pol}.tif')
```

I think this will reduce the possibility of incorrectly locating the RTC-S1 products with `first_pol`